package.json

```diff
@@ -15,7 +15,7 @@
   "dependencies": {
     "async": "^2.6.1",
     "disyuntor": "^3.5.0",
-    "ioredis": "^4.5.1",
+    "ioredis": "^4.28.5",
```

Comment:
This is not a huge change. What's the changelog is saying here?

Reply:
https://github.com/luin/ioredis/compare/v4.5.1..v4.28.5#diff-c275fec9c453c2a42515bc5ab47e30fa4130ff1426aef2f6e000a9a34e122cb8R1

Mostly bug fixes!
Most feature changes are small, I do not see any red or salmon colored flags, PTAL though and lmk if anything stands out to you.